includes/OISException.h

```diff
@@ -74,5 +74,7 @@ namespace OIS
 
 //! Use this macro to handle exceptions easily
 #define OIS_EXCEPT( err, str ) throw( OIS::Exception(err, str, __LINE__, __FILE__) )
+//#define OIS_WARN( err, str ) throw( OIS::Exception(err, str, __LINE__, __FILE__) )
+#define OIS_WARN( err, str ) do {} while(0)

```

Comment:
I'm unsure about the usefulness of this "do nothing" macro as of now. And a "warning" shouldn't be an exception either...